apps/examples/src/examples/inline/InlineExample.tsx

```diff
@@ -23,6 +23,7 @@ export default function InlineExample() {
 					gap: 12,
 				}}
 			>
+				<InlineEditor width={900} height={600} />
```

Comment:
Added another size to this example to highlight these sorts of changes more clearly.